### ISSUE TITLE
Blank screen when starting terminus

### ISSUE THREAD

**Issue by cblake777** [randy3k/Terminus#148]
I ran into a problem attempting to open a Terminus terminal off a fresh install.  If I attempt to use "Open Default Shell in View" from within the sublime text project, all I am left with is a blank tab, the title says "bash" and I am able to type in the empty window, but there is no terminal, it's just like an empty text editor (see attached image).  There are no errors in the console to indicate a failure that I can see.

I *believe* the problem may be related to a file system mounted via fuse (sshfs), but I am not sure of the exact cause.  This problem only happens in a sublime "project" of mine who's files are all located on a remotely mounted file system.  If I attempt to launch Terminus on a standard sublime text window, no problem is encountered.

I was able to work around it by creating a custom command to open the default terminal with the home directory hard coded as the cwd path.  For example:

    [
        {
            "caption": "Terminus: Open Default Shell at Home",
            "command": "terminus_open",
            "args"   : {
                "cwd": "/Users/username"
            }
        }
    ]

<img width="569" alt="Screenshot of Sublime Text (8-14-19, 12-02-54 PM)" src="https://user-images.githubusercontent.com/391509/63036649-7558db80-be8b-11e9-86db-8b9808ee625b.png">

**Comment by randy3k** [comment]
Is there any error message in the console?

**Comment by cblake777** [comment]
No, there is no error message in the console that I have been able to find.

**Comment by yarnball** [comment]
This can happen when you've messed up your system preferences for the shell.

Try going to "Users & Groups" > "Advanced Settings" and make sure you're set to "/bin/bash" or whatever you prefer.

https://rick.cogley.info/img/Cogley-Post-System-Preferences-User-Advanced-Options-for-Shell-Change.jpg

**Comment by vinhmdev** [comment]
I encountered a similar problem on my Macbook Pro 2019 (Intel). After installing Terminus, I simply opened the application, but after opening it, only a black screen appeared, nothing else.

<img width="894" height="586" alt="Image" src="https://github.com/user-attachments/assets/0e43422b-96d8-423f-9c4b-00a148a1a1b8" />

**Comment by elouangrimm** [comment]
@vinhmdev I just reinstalled the Linux version, and I am stuck on that identical page as well. I waited about 40 minutes, still nothing. I wonder what's wrong
